### DISCUSSION TITLE
2024 Virtual Workshop on Single-Particle Cryo-ET in nextPYP

### DISCUSSION THREAD

**Discussion by abartesaghi** [nextpyp/.github#38] [Announcements]
Dear all,

If you are interested in developing data processing skills for in-situ structure determination, we invite you to join the _“2024 Virtual Workshop on Single-Particle Cryo-ET in nextPYP”_ on **Thursday March 7th** and **Friday, March 8th 2024**. 

The workshop is aimed at cryo-ET practitioners that want to learn about state-of-the-art tools implemented in the package nextPYP (https://nextpyp.app/). The first day will feature a webinar covering all aspects of the tomography processing pipeline, from tilt-frame alignment to high-resolution refinement and classification, and on-the-fly data processing. The second day will feature hands-on practicals where participants will follow step-by-step tutorials, ask questions and get help from the nextPYP team. The full agenda is available [here](https://docs.google.com/document/d/1NpMy8piQmpE3KZN-L-GdXJSWOhGZRHk3lHBPVECbePQ).

The [Bartesaghi Lab](https://cryoem.cs.duke.edu/) will provide access to a limited number of instances of nextPYP and the necessary resources to run the tutorials. Participants are also welcome to follow along using their own resources.

If you are interested in attending, fill out the [registration form](https://duke.zoom.us/meeting/register/tJEscu2opz4tEtAb_fCDnwR7Wvv1PoF1CY8b). 

**Registration deadlines**:
- **February 24th, 2024** if participating in the practicals
- M**arch 5th, 2024** if only attending the webinar

Cheers,
Alberto Bartesaghi (Duke University, USA)
On behalf of the nextPYP team

**Comment by alexjnoble** [comment]
Looks great, Alberto! Which version of nextPYP will be used for practicals?

-Alex

**Reply by abartesaghi** [reply]
It will likely be 0.6.x since we expect to release at least one minor maintenance version(s) before the workshop. We will send specific instructions (including the exact version we will use) to all workshop participants the week after the February 24th deadline.

**Reply by abartesaghi** [reply]
FYI, we will be using version [0.6.3](https://nextpyp.app/files/pyp/0.6.3/docs/index.html) for the workshop next week.
